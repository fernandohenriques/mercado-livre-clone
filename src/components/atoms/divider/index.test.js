import React from 'react';
import { shallow } from 'enzyme';
import { render } from '@testing-library/react';

import Divider from './index';

describe('<Divider />', () => {
  it('Snapshot testing', () => {
    const { asFragment } = render(<Divider />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('Never render children', () => {
    const wrapper = shallow(
      <Divider>
        <div>children</div>
      </Divider>
    );

    expect(wrapper.exists()).toBeTruthy();
    expect(wrapper.children()).toHaveLength(0);
  });
});
