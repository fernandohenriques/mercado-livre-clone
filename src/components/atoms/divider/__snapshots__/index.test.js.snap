// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> Snapshot testing 1`] = `
<DocumentFragment>
  <div
    class="divider"
  />
</DocumentFragment>
`;
